### PR TITLE
E1777. OSS Project Green: Coherent specification of review requirement

### PR DESCRIPTION
**Pull request by SrijaGanguly**
1)review_strategy.html.erb : Fixed issue of punctuation, capitalization and assigned info buttons to the ambiguous terms while making the interface more readable.
2)list.html.erb : Fixed the issue with ambiguous display of required and allowed reviews of the user at the review list page.
3)student_review_controller.rb : Fixed the issue of wrong implementation of "Maximum number of submissions per document" with the help of a getter method.
4)review_mapping_controller.rb : fixed the issue of "Maximum number of submissions per document" in collaboration with student_review_controller.rb.

### CODE REVIEW

File: app/models/review_assignment.rb [hunk 1]
```diff
@@ -115,12 +115,11 @@ def reject_by_least_reviewed(contributor_set)
     contributor = contributor_set.min_by {|contributor| contributor.review_mappings.reject {|review_mapping| review_mapping.response.nil? }.count }
     min_reviews = contributor.review_mappings.reject {|review_mapping| review_mapping.response.nil? }.count rescue 0
     contributor_set.reject! {|contributor| contributor.review_mappings.reject {|review_mapping| review_mapping.response.nil? }.count > min_reviews + review_topic_threshold }
-
     contributor_set
   end
 
   def reject_by_max_reviews_per_submission(contributor_set)
-    contributor_set.reject! {|contributor| contributor.review_mappings.reject {|review_mapping| review_mapping.response.nil? }.count >= max_reviews_per_submission }
+    contributor_set.reject! {|contributor| contributor.responses.reject {|response| !response.is_submitted }.count >= max_reviews_per_submission }
```
Comment (Winbobob) [inline app/models/review_assignment.rb]: Why did make this change above?

Comment (SrijaGanguly) [reply app/models/review_assignment.rb]: Issue #965 says maximum number of reviews per submission is supposed to check the number of reviews that have been submitted, but in the original code it was checking the number of reviews assigned to the reviewers instead. These assigned reviews may not have been submitted. In this new code, am checking the count of review responses of a contributor that are submitted and validating it with max_reviews_per_submission ( obviously it shouldn't cross this value either).

**Comment by Winbobob** [comment]
The functionality is good. Need to redo test part.

**Comment by ajcpendyala** [comment]
Created another pull request after fixing conflicts #1315

**Pull request closed without merging**